src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentDay = 40;
+  const totalCost = days * rentDay;
+  const salesOffThreeDays = 20;
+  const salesOffSevenDays = 50;
+  const threeDay = 3;
+  const sevenDay = 7;
```

Comment:
what if we change number to another one - then you will should to rename variable, because it is seven and  three in names 😢 
lets fix names of variables